closed/autoconf/custom-spec.gmk.in

```diff
@@ -130,8 +130,10 @@ ifeq ($(OPENJDK_BUILD_OS), macosx)
   # is not applied.
   export MACOSX_DEPLOYMENT_TARGET := @MACOSX_VERSION_MIN@
 
-  # Set page zero size to 4KB for mapping memory below 4GB.
-  LDFLAGS_JDKEXE += -pagezero_size 0x1000
+  ifneq ($(OPENJDK_TARGET_CPU), aarch64)
```

Comment:
While you're here, please replace the reference to `OPENJDK_BUILD_OS` above with `OPENJDK_TARGET_OS` (see the Windows example further above).

Reply:
Updated based on the comment.

Reply:
Updated `OPENJDK_BIULD_OS` lines in Copy.gmk.